src/main/java/us/kbase/auth/AuthException.java

```diff
@@ -0,0 +1,28 @@
+package us.kbase.auth;
+
+import java.lang.Exception;
+
+/** An exception thrown when an auth error occurs. */
+public class AuthException extends Exception {
+	
+	// TODO CODE could add the various fields from the auth server so that services can
+	//           react differently to different error codes
+
+	private static final long serialVersionUID = 1L;
```

Comment:
Is this used anywhere?

Reply:
it's for serialization: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html

Reply:
👍